Scripts/githubLogParser.py

```diff
@@ -0,0 +1,113 @@
+import requests
+import json
+import csv
+from collections import defaultdict
+from pprint import pprint
+import datetime
+
+def employee2github(employeeFile, ghUsername, ghToken, ldapResource):
+    employeesInfo = {}
+
+    while True:
+        employee = employeeFile.readline()
+        if employee != 0:
+            print(f'Trying user {employee}')
+            ldapsearch = f'{ldapResource}'
+            res = requests.get(ldapsearch, auth=(ghUsername, ghToken))
```

Comment:
I will create a JSON file the results I get from those calls (probably host them in a AWS S3 to learn something new, and simulate the request or to move a little faster as a testFile locally. Maybe you have a better suggestion on how to handle it? This specific script is straight forward, but I want to add one that uses LDAP service and that's a little different. I still need to clean up how that code looks like

Reply:
I would expect the GitHub part to be easier to clone, however it depends because it might require setting up repos and a lot of other setup/config, so yes, maybe it would be more practical if you can provide the json outputs for that. 

Ldap seems less intuitive to me, afaik I don't have an ldap account / service so for that I would need some help as well so I can run this too, or is it something we can mock out? Similarly as we're effectively doing with github and you providing json response files?

Thanks